generate-release-cycle.py

```diff
@@ -0,0 +1,131 @@
+"""
+Read in a JSON and generate two CSVs and a Mermaid file
+"""
+from __future__ import annotations
+
+import csv
+import datetime as dt
+import json
+
+MERMAID_HEADER = """
+gantt
+    dateFormat  YYYY-MM-DD
+    title       Python release cycle
+    axisFormat  %Y
+""".lstrip()
+
+MERMAID_SECTION = """
+    section Python {cycle}
+    {release_status}               :{mermaid_status}   python{cycle}, {release_date},{eol}
+"""  # noqa: E501
+
+MERMAID_STATUS_MAPPING = {
+    "features": "",
+    "bugfix": "active,",
+    "security": "done,",
+    "end-of-life": "crit,",
+}
+
+CSV_HEADER = (
+    "Branch",
+    "Schedule",
+    "Status",
+    "First release",
+    "End-of-life",
```

Comment:
Hyphens are used in adjective, for a column heading that’s a noun, we’d have `End of life`

Reply:
```suggestion
    "End of life",
```

Reply:
Updated!